plugins/org.jkiss.dbeaver.ext.oceanbase/src/org/jkiss/dbeaver/ext/oceanbase/model/plan/OceanbasePlanAnalyzer.java

```diff
@@ -109,14 +114,12 @@ public DBPDataSource getDataSource() {
         return this.dataSource;
     }
 
-    @NotNull
     @Override
-    public DBCPlan planQueryExecution(@NotNull DBCSession session, @NotNull String query, @NotNull DBCQueryPlannerConfiguration configuration)
+    public DBCPlan planQueryExecution(DBCSession session, String query, DBCQueryPlannerConfiguration configuration)
```

Comment:
You removed all NotNull annotations in this class. Why? They are very helpful.

Reply:
I mistakenly think this is useless... I'll fix it now and add them back.